endpoints/get_address_transactions.py

```diff
@@ -60,3 +63,42 @@ async def get_transactions_for_address(
     return {
         "transactions": tx_list
     }
+
+@app.get("/addresses/{kaspaAddress}/transactions/v2",
+         response_model=List[TxModel],
+         response_model_exclude_unset=True,
+         tags=["Kaspa addresses transactions"])
+async def get_transactions_for_address_v2(
+        kaspaAddress: str = Path(
+            description="Kaspa address as string e.g. "
+                        "kaspa:pzhh76qc82wzduvsrd9xh4zde9qhp0xc8rl7qu2mvl2e42uvdqt75zrcgpm00",
+            regex="^kaspa\:[a-z0-9]{61}$"),
+        limit: int = Query(
+            description="The number of records to get",
+            ge=1,
+            le=500,
+            default=50),
+        offset: int = Query(
+            description="The offset from which to get records",
+            ge=0,
+            default=0),
+        fields: str = "",
+    ):
+    """
+    Get all transactions for a given address from database.
+    And then get their related full transactiond data
+    """
+
+    async with async_session() as s:
+        # Doing it this way as opposed to adding it directly in the IN clause
+        # so I can re-use the same result in tx_list, TxInput and TxOutput
+        txWithinLimitOffset = await s.execute(select(TxAddrMapping.transaction_id)
```

Comment:
If you don't mind use lowercase with underscores. Trying to follow PEP8

https://peps.python.org/pep-0008/#function-and-variable-names

Reply:
Resolved with ff606f4